assets/scripts/script.js

```diff
@@ -9,5 +9,10 @@
       }, 600);
     });
 
+    if (typeof $.fn.lightGallery === 'function' && $('body').hasClass('woocommerce') && $('.images').length > 0) {
+      $('.images').on('onAfterAppendSubHtml.lg', function (event, index) {
+        $('.lg-sub-html').append('<span>Bild zeigt evtl. Zubehör. Lieferumfang beachten!</span>');
```

Comment:
Shouldn't this use the actual string translation?
https://github.com/netzstrategen/woocommerce-image-disclaimer/blob/7afacfeeaca608bbf987252765b58802861658df/languages/image-disclaimer-de_DE.po#L26-L27

Reply:
I did not translate it because in WordPress 5.0 they added [a new i18n support for JS](https://make.wordpress.org/core/2018/11/09/new-javascript-i18n-support-in-wordpress/). It seems to be better in term of performances.

Since we are using WP 4.9.8 but will soon bump to 5.x (see updates branch), I think we can add this as part of the updates instead of code it now and change it in a while. 🙂